joint_state_publisher/joint_state_publisher.py

```diff
@@ -13,6 +13,7 @@
 # ROS 2 imports
 import rclpy
 import rclpy.parameter
+from rclpy.parameter import Parameter
```

Comment:
Nit; I'd prefer not to do this and just use the fully-qualified `rclpy.parameter.Parameter` below.